### PR TITLE
Add information

### PR DESCRIPTION
**Pull request by mpivaa**
Add our week and project planning

@yoshioteruia please review english.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
-######Things you need to know before starting:
+#####Things you need to know before starting:
 
 * We are a flat organization with very high standards
 * We work mostly with Ruby on Rails apps, on Heroku infrastructure.
 * Write nice code is mandatory, we follow this convention: http://github.com/tracersoft/guides
 * You will work with a small team, usually 3 devs. So, you have to be good on communication and acceptable with feedbacks!
-* eXtreme Programming: http://www.desenvolvimentoagil.com.br/xp/
-* Red, Green, Refactor: you must know or learn TDD.
+* Our methodology is based in eXtreme Programming and Scrum: http://www.desenvolvimentoagil.com.br/xp/
```
Comment (yoshioteruia) [inline README.md]: Alterar based in por based on

File: README.md [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
+* Red, Green, Refactor: you must know or learn **TDD**.
 * We have free fridays. Clients usually test features, which are developed over the week, on fridays. But "free" does not mean like a "lazy day", it is a day in which we can learn new things, develop side projects or contribute to open source projects.
+* Our **daily meeting** is done between 10am and 12am (always GMT -2).
+* Our day starts between 8am and 10am and ends upto between 5pm and 7pm. Remote people must be online between 10am and 5pm.
```
Comment (yoshioteruia) [inline README.md:17]: Our day starts between 8am and 10am and ends between 5pm and 7pm. Remote people must be online between 10am and 5pm.

**Comment by phillipp-oliveira** [comment]
:+1:

**Comment by mpivaa** [comment]
:+1:

**Pull request merged**